man/man8/zfs.8

```diff
@@ -756,8 +756,9 @@ This property can also be referred to by its shortened column name, \fBvolblock\
 .ad
 .sp .6
 .RS 4n
-The amount of \fBreferenced\fR space written to this dataset since the
-previous snapshot.
+The amount of space \fBreferenced\fR by this dataset, that was written since the previous snapshot
+(i.e. that is not referenced by the previous snapshot)
+space written to this dataset since the previous snapshot.
```

Comment:
This looks like a mistake in the original patch.  This last line, 761, should have been removed.